charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -175,8 +175,6 @@ alfresco-content-services:
       repository: quay.io/alfresco/alfresco-dbp-repository
       tag: "0.4-SNAPSHOT"
     environment:
-      # The SYNC_SERVICE_URI should contain the details of the ingress host and port.
-      SYNC_SERVICE_URI: "<ingresscontrollerurl>/syncservice"
```

Comment:
We'll still need to supply this value.

Reply:
This property is defined in the ACS chart configuration, doesn't need to exist here anymore.

Reply:
We still need to set the URI based on the domain being used for ingress into the cluster.

Reply:
Reverted the change.